MSAL/test/integration/native_auth/end_to_end/sign_in/MSALNativeAuthSignInUserNameAndPasswordEndToEndTests.swift

```diff
@@ -94,4 +83,32 @@ final class MSALNativeAuthSignInUsernameAndPasswordEndToEndTests: MSALNativeAuth
         XCTAssertNotNil(signInDelegateSpy.result?.idToken)
         XCTAssertEqual(signInDelegateSpy.result?.account.username, username)
     }
+    
+    func test_signInAndSendingIncorrectPasswordResultsInError() async throws {
+        guard let sut = initialisePublicClientApplication(clientIdType: .code), let username = getSignInUsernamePassword() else {
```

Comment:
Should this be `.password`?

Reply:
It can be anything to be precise. A user will be required to signIn based how they signUp, not based on the client Id in use. 
But let me remove the clientIdType for correctness